src/modules/ie-objects/helpers/convert-string-to-search-terms.ts

```diff
@@ -19,7 +19,10 @@ export function convertStringToSearchTerms(searchQuery: string): string[] {
 }
 
 export function convertNodeToSearchTerms(node: Expression): string[] {
```

Comment:
we should check that this function also returns the whole literal string als a term in a test case:
```
		filters: [
			{
				field: IeObjectsSearchFilterField.QUERY,
				operator: Operator.CONTAINS,
				value: 'De grote -)( test',
			},
		],		
```

response should contain:
```
{
	...
	searchTerms: ["De grote -)( test"],
}
```